index.html

```diff
@@ -51,10 +54,10 @@
         </div>
     </div>
     <div class="header-bottom row">
-        <div class="col-mobile-100">
-            <img class="col-large-tablet-hide col-desktop-hide" src="image/mobile-header.jpg" alt="">
+        <div class="header-bottom--height col-mobile-100 col-large-tablet-hide col-desktop-hide">
```

Comment:
damian to jest dalej zle, tym razem to ty mi wytlumaczysz dlaczego ;), wszystkie klasy header-bottom--height nie sa zgodne z BEM.

Ponizej zrobilem prawidlowo, a tutaj byk ;)